circle.yml

```diff
@@ -26,4 +30,6 @@ test:
     - gem install --no-rdoc brakeman; bundle exec brakeman --exit-on-warn .
     - gem install --no-rdoc ruby_audit; bundle exec ruby-audit check
     - gem install --no-rdoc bundler-audit; bundle-audit update; bundle-audit check
+    - gemnasium auth login --token $GEMNASIUM_TOKEN
```

Comment:
Can you make this a one liner with the autoupdate script? e.g. `gemnasium auth login && bash autoupdate.sh`?